### PR TITLE
Add measurement for Enterprise_Index changelogs

### PR DESCRIPTION
**Pull request by mpchadwick**
Implementation of enhancement documented in #25.

cc @toddbc

### CODE REVIEW

File: app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+class SomethingDigital_InfluxDb_Model_Measurement_Changelog
+    extends SomethingDigital_InfluxDb_Model_Measurement_Abstract
+    implements SomethingDigital_InfluxDb_Model_MeasurementInterface
+{
+    public function send()
+    {
+        $metadatas = Mage::getModel('enterprise_mview/metadata')->getCollection();
+        $data = array();
+        foreach ($metadatas as $metadata) {
+            $data[] = $this->line($metadata);
```
Comment (philwinkle) [inline app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php]: This is essentially a map:

```php
$data = array_map($metadatas, array($this,'line'));
```

File: app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    protected function line($metadata)
+    {
+        $changelog = Mage::getModel('enterprise_index/changelog', array(
+            'connection' => $this->readCon,
+            'metadata' => $metadata,
+        ));
+
+        return 'changelog,name=' . $metadata->getChangelogName() .
```
Comment (philwinkle) [inline app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php]: Nit: No me gusta concatenation. I <3 `sprintf`. But this will do.

**Comment by philwinkle** [comment]
LGTM

File: app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php (hunk 1 above)
Comment (mpchadwick) [inline app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php]: Good call. That's a lot cleaner.

File: app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php (hunk 2 above)
Comment (mpchadwick) [inline app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php]: A lot of the measurements are using string concatenation.

https://github.com/sdinteractive/SomethingDigital_InfluxDb/blob/develop/app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Cron.php#L54-L58

It is a bit ugly, though.

File: app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php (hunk 1 above)
Comment (philwinkle) [inline app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php]: I had the arguments reversed for `array_map` but yeah, I think it's prettier without being obtuse.

**Pull request merged**